integration-tests/src/__screenshots__/Example-tests.js

```diff
@@ -2,17 +2,15 @@ import Example from '../Example';
 import React from 'react';
 
 describe('Example', () => {
-
   it('basic components work', () => {
-    return (
-      <Example>This is some text</Example>
-    );
+    return <Example>This is some text</Example>;
   });
 
-  it('components with custom dimensions work', () => {
-    return (
-      <Example>This is some text</Example>
-    );
-  }, [320, 768]);
-
+  it(
+    'components with custom dimensions work',
```

Comment:
@thomashuston Do you know why it would have moved this down a line button didn't with line 5 above?  Looking at it more closely, I'm guessing it may have been due to the width argument being included - and if it has one argument **starting** on a new line, it splits them all like this..